src/Components/PackagingOrders/BookmarkToolbar.jsx

```diff
@@ -0,0 +1,59 @@
+function BookmarkToolbar() {
+  return (
+    <>
+      <span className="block content-center pb-1 pt-3 text-lg font-bold text-gray-700">
+        즐겨찾기
+      </span>
+      <div className="flex flex-row justify-around py-1">
+        <div className="flex flex-row">
+          <span className="block content-center px-3 text-sm text-gray-500">
+            추가하기
+          </span>
+          <button
+            type="button"
+            className="inline-flex h-10 w-10 items-center justify-center rounded-full border-none bg-blue-600 outline-none hover:bg-blue-700 active:bg-blue-600"
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="14px"
+              fill="#fff"
+              className="inline"
+              viewBox="0 0 512 512"
+              alt="add-bookmark-icon"
+            >
+              <path
+                d="M467 211H301V45c0-24.853-20.147-45-45-45s-45 20.147-45 45v166H45c-24.853 0-45 20.147-45 45s20.147 45 45 45h166v166c0 24.853 20.147 45 45 45s45-20.147 45-45V301h166c24.853 0 45-20.147 45-45s-20.147-45-45-45z"
+                data-original="#000000"
+              />
+            </svg>
+          </button>
+        </div>
+        <div className="flex flex-row">
+          <span className="block content-center px-3 text-sm text-gray-500">
+            가져오기
+          </span>
+          <button
+            type="button"
+            className="inline-flex h-10 w-10 items-center justify-center rounded border-none bg-yellow-400 outline-none hover:bg-yellow-600 active:bg-yellow-500"
+          >
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="16px"
+              fill="#fff"
+              className="inline"
+              viewBox="0 0 24 24"
+              alt="bookmark icon"
+            >
```

Comment:
namespace 를 명시해주셨군여
해당 자료 관련해서 조사해 봤던 [자료](https://www.w3.org/TR/xml-names/#concepts) 남깁니다. 나중에 천천히 보시면 좋을것 같습니다.👍

className="inline" 속성에 대해서는 필요없을 것 같습니다. 
관련된 [자료](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/display) 남겨 놓습니다.

width 속성과 fill 속성을 tailwind 와 합쳐서 최종적으로 아래와 같이 나타낼 수 있을것 같습니다.😄
```suggestion
            <svg
              xmlns="http://www.w3.org/2000/svg"
              className="w-3.5 text-white"
              viewBox="0 0 24 24"
              alt="bookmark icon"
            >
```

Reply:
수보님 리뷰 감사합니다!
리뷰 남겨주신 부분중에서 ```className```이 혼란을 야기하고 있는 것 같습니다. 
먼저, SVG 내의 모든 속성은 SVG 에서 제공하고 있는 속성값들을 사용하고 있는 것으로, tailwind를 사용하고 있지 않습니다. 
다만 원래 ```class```로 명시됐어야 했던 속성이 린트 규칙에 따라 ```className```으로 수정되면서 마치 tailwind 가 적용 되고 있는것 처럼 보이게 된 것 같습니다. (원래는 className을 사용했지만 이미 한참전에 더 이상 지원되지 않는 것으로 나옵니다. [참고](https://github.com/facebook/react/issues/6211))

결론적으로, fill 속성은 그대로 내버려 두지만 현재 전반적인 코드 스타일에 맞춰 #fff 에서 #ffffff 로 수정하고 
SVG 내에서는 ```className``` 대신 ```class``` 를 사용하되, 현재 inline 의 경우 디폴트 속성이므로 삭제하는 방향으로 수정 해보도록 하겠습니다~! 😊

Reply:
엇 그렇군여! 
저도 몰랐던 내용이였네여 참고 자료 잘 확인했습니다.👍
몰랐던 내용 잘 배워갑니다.🙇‍♂️